packages/kit/src/exports/vite/index.js

```diff
@@ -59,9 +59,27 @@ const enforced_config = {
 	root: true
 };
 
-/** @return {import('vite').Plugin[]} */
-export function sveltekit() {
-	return [...svelte(), kit()];
+/** @return {Promise<import('vite').Plugin[]>} */
+export async function sveltekit() {
+	const svelte_config = await load_config();
+	/** @type {import('@sveltejs/vite-plugin-svelte').Options} */
+	const vite_plugin_svelte_options = svelte_config.vitePlugin || {};
+
+	// Default hydratable to true
```

Comment:
should `hydratable: true` go before `...vite_plugin_svelte_options.compilerOptions`? If this is correct as is, then the comment should probably be changed from "Default" to "force" and we should warn if we override the user's config just as we do with `enforced_config`

Reply:
It shouldn't be overriding it since it does a `vite_plugin_svelte_options.compilerOptions?.hydratable == null` check before. We could skip the check and make it go before the `...` and it should work the same too.

Reply:
updated to put `hydratable: true` before `...`. I also found that I was doing the whole flow wrong 🙈